kubernetes/machine-class.yaml

```diff
@@ -42,6 +42,9 @@ providerSpec:
 secretRef: # Secret pointing to a secret which contains the provider secret and cloudconfig
   namespace: default  # Namespace
   name: test-secret # Name of the secret
+# capacityReservation: # Optional - allows targeting of an AWS Capacity reservation or an AWS Resource Group containing reservations. Use either a reservation id or group arn - but not both.
```

Comment:
If this is an either/or field, it would probably make sense to check this at the time of [validation of the spec here ](https://github.com/gardener/machine-controller-manager-provider-aws/blob/master/pkg/aws/apis/validation/validation.go#L59)right? You can probably have a new helper method to validate the same.

Reply:
Ah yes that makes more sense. Done.